### PR TITLE
Add travis shared config, tab to question mark icon

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/9883

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -46,7 +45,7 @@ const config = [
     },
     devtool: (ENV !== 'production') ? 'source-map' : '',
     resolve: resolveJS(ENV, PATHS),
-    externals,
+    // externals: externalJS(ENV, PATHS),
```
Comment (emteknetnz) [inline webpack.config.js:48]: Commenting this out was required to resolve the 'jQuery is not defined' issue that the webpack loader was having.  This webpack config was originally intended for use within the CMS where jQuery is globally defined.  We're don't having it globally defined in login-forms, instead it's imported via npm

File: themes/login-forms/templates/SilverStripe/Forms/CheckboxField.ss [hunk 2]
```diff
@@ -1 +1 @@
-<input $getAttributesHTML('title', 'data-toggle') />
+<input $getAttributesHTML('title') />
```
Comment (bergice) [inline themes/login-forms/templates/SilverStripe/Forms/CheckboxField.ss:1]: No need to include `data-toggle` here.

```suggestion
<input $getAttributesHTML('title', 'data-toggle') />
```

**Review by bergice — CHANGES_REQUESTED** [review]
LGTM, just want a small change.

Won't merge, as I'm assuming the travis stuff is still WIP.

File: themes/login-forms/templates/SilverStripe/Forms/CheckboxField.ss (hunk 2 above)
Comment (emteknetnz) [reply themes/login-forms/templates/SilverStripe/Forms/CheckboxField.ss:1]: :-) I think I removed it and your recommendation is to add it back in?

Comment (bergice) [reply themes/login-forms/templates/SilverStripe/Forms/CheckboxField.ss:1]: Including it as an argument will exclude it from being rendered. Check the function definition.

Comment (emteknetnz) [reply themes/login-forms/templates/SilverStripe/Forms/CheckboxField.ss:1]: yeah but we're no longer adding data-toggle - was removed from the framework pr remember

**Pull request merged**